recipe/deploy/release.php

```diff
@@ -18,21 +18,13 @@
         return [];
     }
 
-    $keepReleases = get('keep_releases');
-    if ($keepReleases === -1) {
-        $data = run('cat .dep/releases_log');
-    } else {
-        $data = run("tail -n " . ($keepReleases + 5) . " .dep/releases_log");
-    }
+    $releaseLogs = array_map(function ($line) {
+        return json_decode($line, true);
+    }, explode("\n", run('cat .dep/releases_log')));
 
-    $releasesLog = [];
-    foreach (explode("\n", $data) as $line) {
-        $metainfo = json_decode($line, true);
-        if (!empty($metainfo)) {
-            $releasesLog[] = $metainfo;
-        }
-    }
-    return $releasesLog;
+    return array_filter($releaseLogs, function ($release) {
+        return ! empty($release);
```

Comment:
No need for this (I guess?) because array_filter by defaults removes everything which evaluates to false

Reply:
Updated as suggested.